ETC.tsv

```diff
@@ -2551,7 +2551,7 @@ ETC_20150317_002550	{memo X}Obtained Mayvern's key!
 ETC_20150317_002551	{memo X}Naktis' subordinates are being too disrupting!
 ETC_20150317_002552	{memo X}Using a key of Mayvern..
 ETC_20150317_002553	Experimental Mesyum
-ETC_20150317_002554	$Sprinkling the Acidic Solution
+ETC_20150317_002554	$Pouring the Acidic Solution
```

Comment:
I suppose, but it's kind of hard to 'sprinkle' something from a bottle, unless that bottle had a cover with small holes on it.